src/components/views/OnlineModView.vue

```diff
@@ -1,71 +1,82 @@
-                            class="input"
-                            type="text"
-                            placeholder="Search for a mod"
-                            autocomplete="off"
-                        />
-                    </div>
-                    <div class="input-group margin-right">
-                        <label for="thunderstore-sort">Sort</label>
-                        <select v-model="sortingStyleModel"
-                                id="thunderstore-sort"
-                                class="select select--content-spacing margin-right margin-right--half-width"
-                        >
-                            <option v-for="(key) in getSortOptions()" v-bind:key="key">{{key}}</option>
-                        </select>
-                        <select v-model="sortingDirectionModel"
-                                class="select select--content-spacing"
-                                :disabled="sortingStyleModel === 'Default'"
-                        >
-                            <option v-for="(key) in getSortDirections()" v-bind:key="key">{{key}}</option>
-                        </select>
-                    </div>
-                    <div class="input-group">
-                        <div class="input-group input-group--flex">
-                            <label for="thunderstore-category-filter">Additional filters</label>
-                            <button
-                                id="thunderstore-category-filter"
-                                class="button"
-                                @click="$store.commit('openCategoryFilterModal')"
-                            >
-                                Filter categories
-                            </button>
+    <div class="split-pane" :class="[{'split-pane--with-active-second-pane': previewMod !== null}]">
+        <div id="online-view">
+            <div id="controls">
+                <div class="inherit-background-colour non-selectable">
+                    <div class="is-shadowless is-square">
+                        <div class="no-padding-left card-header-title">
+                            <div class="input-group input-group--flex margin-right">
+                                <label for="thunderstore-search-filter">Search</label>
+                                <DeferredInput
+                                    v-model="thunderstoreSearchFilter"
+                                    id="thunderstore-search-filter"
+                                    class="input"
+                                    type="text"
+                                    placeholder="Search"
+                                />
+                            </div>
+                            <div class="input-group">
+                                <div class="input-group input-group--flex">
+                                    <label for="thunderstore-category-filter">&nbsp;</label>
+                                    <button
```

Comment:
I like how using the modal button here instead of the selects clean up the UI 👍Outside of the scope here but we should consider something similar to installed mod view?

Reply:
We should at some point. It would be nice to add the panel to the installed list too and would make sense to do it for a release following this one.

I'm not doing it currently as I think we can do this incrementally and the online section is the bigger benefit IMO. We can also gauge reactions more this way without a large scale overhaul.

For the installed one we'd instead need to read from the cache, and some may not even have a README if they were locally imported. It's a bit of a logic change and I think we can address it separately.